module/Decision/src/Decision/View/Helper/Organ.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
I can't find any occurence of the `findOrgan` in the code. So I would guess not.